packages/shared/lib/db/config.ts

```diff
@@ -9,7 +9,8 @@ const config: { development: Knex.Config<any>; production: Knex.Config<any> } =
             user: process.env['NANGO_DB_USER'] || 'nango',
             database: process.env['NANGO_DB_NAME'] || 'nango',
             password: process.env['NANGO_DB_PASSWORD'] || 'nango',
-            ssl: process.env['NANGO_DB_SSL'] != null && process.env['NANGO_DB_SSL'].toLowerCase() === 'true' ? { rejectUnauthorized: false } : undefined
+            ssl: process.env['NANGO_DB_SSL'] != null && process.env['NANGO_DB_SSL'].toLowerCase() === 'true' ? { rejectUnauthorized: false } : undefined,
+            statement_timeout: 60000
```

Comment:
Note: this would affect all queries including migrations or activity logs clearnup.

@bastienbeurier @khaliqgant I am happy to reevalutate this value. do we wan to be more/less aggresive?

Reply:
I think 60s is good to start with. Worst case some requests fail and customers let us know, right? 

Probably worth have a graph with the count of these timeouts, to measure the scale of timeout-related failures, if not too complex.

Reply:
I don't think the database monitoring instruments every query, it is more about aggregated metrics.
We can add a graph based on the server query results but it won't take any account any queries from batchSave/etc... since the runner is not instrumented yet

Reply:
I worry about the activity log cleanup query. How long were you seeing migrations taking on prod @TBonnin?

Reply:
last migration was very fast since since it was a create index CONCURRENTLY.

The activity log cleanup actually takes 50s. https://app.datadoghq.com/databases/queries?query=command%3Adelete&dbPanels=%5B%7B%22t%22%3A%22dbQueryPanel%22%2C%22qs%22%3A%229e474a577ecd33e2%22%2C%22dbType%22%3A%22PostgreSQL%22%7D%5D&dbPlanSignature=4a8bda378c0b320b&dbType=PostgreSQL&start=1702372328926&end=1702977128926&paused=false

@Bastien and I thought it might be good to run it every hour so it only takes a few seconds. I made the change here [add219e](https://github.com/NangoHQ/nango/pull/1434/commits/add219e611989249a4451c288bc7decc740fd078)